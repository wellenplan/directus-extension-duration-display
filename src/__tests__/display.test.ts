import { mount } from "@vue/test-utils";
import DisplayComponent from "./../display.vue";

const props = [
  {},
  {
    value: 1,
  },
  {
    value: 60,
  },
];

describe("duration", () => {
  describe("display", () => {
    it.each(props)("should render %s", (props: any) => {
      const wrapper = mount(DisplayComponent, {
        props: props,
      });
      expect(wrapper.html()).toMatchSnapshot();
    });
  });
});
