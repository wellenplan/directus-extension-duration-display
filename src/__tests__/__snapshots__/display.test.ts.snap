// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`duration display should render { value: 1 } 1`] = `"<span>1 second</span>"`;

exports[`duration display should render { value: 60 } 1`] = `"<span>1 minute</span>"`;

exports[`duration display should render {} 1`] = `"<span>0 seconds</span>"`;
